### DISCUSSION TITLE
Ubuntu not having the latest

### DISCUSSION THREAD

**Discussion by tarsil** [containers/podman#22867] [General]
When we run `sudo apt install podman -y` as per instructions, it will install the version 3.4.4 and not the 5.1.0. This is annoying since we can't use the latest features on a ubuntu/debian machine.  Homebrew already has it but its hard to make it work on a Debian/Ubuntu/Mint machine. hen can we expect the latest to be available for these?

**Comment by afbjorklund** [comment]
* https://github.com/containers/podman/discussions/17362

**Reply by tarsil** [reply]
Thank you @afbjorklund but this still doesn't answer the question. This is frustrating. Even MacOS and Windows have a more up to date version and we can't have it on the most used disyris out there.

**Reply by afbjorklund** [reply]
Recent versions of Podman Desktop allows you to run in a VM (with Fedora), to get the same level of integration as is available for Mac and Win... But you have to enable it yourself, if you want it.

* https://github.com/containers/podman-desktop/discussions/5762

**Reply by tarsil** [reply]
> Recent versions of Podman Desktop allows you to run in a VM (with Fedora), to get the same level of integration as is available for Mac and Win... But you have to enable it yourself, if you want it.
> 
> * [Using Podman Desktop on Linux with newer version in Podman Machine podman-desktop#5762](https://github.com/containers/podman-desktop/discussions/5762)

That defeats the whole purpose no? I have a Debian based OS, actually, two and the maximum I can get is the version 3.4.4 and it's known for years. When I use brew to install which is ok, I can't get it to work because it complains about the podman.socket which after numerous days searching, there is no Linux solution to workaround that. 

This makes people not moving to Podman 100% and slows down adoption by a lot

**Reply by afbjorklund** [reply]
> That defeats the whole purpose no?

It depends on what the purpose is...

https://docs.docker.com/desktop/faqs/linuxfaqs/#why-does-docker-desktop-for-linux-run-a-vm

* https://github.com/containers/podman-desktop/issues/3068

But you can still build it from source.

**Reply by afbjorklund** [reply]
> When I use brew to install which is ok, I can't get it to work because it complains about the podman.socket

This sounds like a brew issue, it shouldn't be more than `systemctl enable podman.socket` to start the daemon

**Reply by tarsil** [reply]
Thank you for this. So, building from the source complains about the the narvark (forgot the full name here). Then to uninstall the purge doesn't work anymore. 

The brew doesn't create the podman socket

**Reply by afbjorklund** [reply]
You also have to build all the dependencies for it.

https://podman.io/docs/installation#building-from-source

**Reply by tarsil** [reply]
Of course but without it you can't even built it

**Reply by tarsil** [reply]
@afbjorklund i was able to finally make a build but it's complaining about the conmom but that will be addressed later. 

A great way for podman to help solving this issue would be by proving a bash script that basically does the build and setup in one go. At least a lot of people could simply reuse it and go from there without relying on the package manager

**Reply by afbjorklund** [reply]
There was a Dockerfile instead, since you also need the compilers etc

**Reply by tarsil** [reply]
But the Dockerfile doesn't natively install podman as majority of the Debian/Ubuntu/similar users need. A simple script would literally do the same job and would be probably the best case. I don't know why the launchpad doesn't have the latest of podman. But even better would be for podman to have a similar download process like docker has (download.podman.io/....). 

I fail to understand why if they call it a proper drop in replacement this is not happening
